src/features/images/imagesSlice.js

```diff
@@ -180,8 +182,13 @@ export const imagesSlice = createSlice({
     },
 
     setDeleteImagesAlertOpen: (state, { payload }) => {
-      state.deleteImagesAlertOpen = payload;
```

Comment:
Debating the name of this function, curious for any suggestions here in case this isnt clear

Reply:
Good question... I was a little confused by [this line](https://github.com/tnc-ca-geo/animl-frontend/blob/d95cfc47c37f82dadfea42028bff15c97b7fb6a0/src/features/images/ImagesTableRow.jsx#L284) in ImageTableRow.jsx when I first came across it because the `false` payload makes it seem like we're closing the Alert.

I would probably consolidate `setDeleteImagesAlertOpen` and `setDeleteImagesAlertClose` into a single `setDeleteImagesAlert` action reducer that accepts an object payload with a shape of `{ open: <boolean>, type: <'by-filter' | 'by-id> }` so when we're dispatching from components it it's a bit more explicit, e.g., `dispatch(setDeleteImagesAlert({ open: true, type: 'by-id' }))`.

Reply:
Using 2 booleans instead of having type be a string until we have more deleting methods